main.go

```diff
@@ -175,13 +208,19 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-
 	// create the faucet instance
-	f := faucet{
-		signer:     &signer,
-		authTypes:  authTypes,
-		waitPeriod: waitPeriod,
-		storage:    storage,
+	f := faucet.Faucet{
+		Signer:     &signer,
+		AuthTypes:  authTypes,
+		WaitPeriod: waitPeriod,
+		Storage:    storage,
+	}
+	var s *stripehandler.StripeHandler
+	if stripeKey != "" && stripeWebhookSecret != "" {
+		s = stripehandler.NewStripeClient(stripeKey, stripePriceId, stripeWebhookSecret, stripeMinQuantity, stripeMaxQuantity, stripeDefaultQuantity, &f, storage)
+		log.Info("Stripe enabled")
```

Comment:
We don't use initial capital letters on log events. Moreover, if you want to print a log, please add more info.